core/templates/pages/topic-viewer-page/topic-story-section/topic-story-section.component.html

```
@@ -0,0 +1,42 @@
+<section class="topic-story-section"
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)